### ISSUE TITLE
Identify and eliminate SPD seed dependence

### ISSUE THREAD

**Issue by danbraunai** [goodfire-ai/spd#14]
**Migrated from:** goodfire-ai/spd-gf#50
**Original author:** @leesharkey

---

Even when a set of hyperparameters gets some runs right on some seeds, some runs fail. There is therefore still some seed dependence (on resid-mlp 3 layer especially). 

SPD should at very least be robust to choice of seed. Something is wrong if it is sensitive to seed. But it feels fixable. 

This may involve investigating particular choices in the SPD algorithm, including sigmoid shape; CI network architecture/activation function/inputs; which losses to include and which to exclude, etc. 

In general, we will want to do full evals sweeps with any given setup you choose to explore. This will therefore likely be pretty compute intensive.

**Comment by danbraunai** [comment]
The solution to this issue may come from many things - CI functions, initialization, loss functions. I don't think this is a good issue to pick up in itself rather than it falling out of one of the other analyses.

**Comment by danbraunai** [comment]
Closing. This issue is too similar to "improve spd" to be useful.
